mandatory/3-function-output.js

```diff
@@ -1,14 +1,21 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/*Add comments to explain what this function does. You're meant to use Google!
+
+ This function it's going to take a random number from 0 to 1(excluding ) and it's going to multiply that
+number by 10 */
 function getRandomNumber() {
   return Math.random() * 10;
-}
+}  
 
-// Add comments to explain what this function does. You're meant to use Google!
+/*Add comments to explain what this function does. You're meant to use Google!
+
+The concat method joins 2 or more strings together,without  changing the existing string but 
```

Comment:
Correct, but try to make it simpler, e.g. "Combines two words without any modifications"